### PR TITLE
Working BE for soccer (scrapping)

### PR DESCRIPTION
**Pull request by blakeparkinson**
I will do the rendering tomorrow

### CODE REVIEW

File: public/javascripts/quiz.js [hunk 1]
```diff
@@ -3,6 +3,13 @@ var starters,bench, roster,
     stop_counter = false;
     team_container = $('.team-container'),
     answer_container = $('.answer-container');
+if (typeof(roster) != undefined){
```
Comment (kelseyedmodo) [inline public/javascripts/quiz.js]: curious - when is it best to do a "typeof" check vs. a "length" check?

**Comment by kelseyedmodo** [comment]
Is broken for Charlotte Hornets (TypeError: Cannot read property 'starters' of undefined)
but worked for the Atl. Hawks.

didn't look at why charlotte is different. fix that and g2g.

File: public/javascripts/quiz.js (hunk 1 above)
Comment (blakeparkinson) [inline public/javascripts/quiz.js]: you cant check length of an undefined variable. You would check length of something that will never be defined. Like a jquery dom pull or something

**Comment by blakeparkinson** [comment]
looks good :umbrella:

**Pull request merged**